tst/testinstall/range.tst

```diff
@@ -1,16 +1,72 @@
-#@local g,ranges,x,y,z,a,f
+#@local TestPrintRangeRep,g,ranges,x,y,z,a,f
 gap> START_TEST("range.tst");
 
 #
 gap> [0..0];
 [ 0 ]
+gap> [0..1];
+[ 0, 1 ]
 gap> [0..-1];
 [  ]
 gap> [-5..5];
 [ -5 .. 5 ]
 gap> [-5,-3..5];
 [ -5, -3 .. 5 ]
 
+#
+gap> TestPrintRangeRep:=function(r)
+>    Assert(0, IsRangeRep(r));
+>    Print("ViewObj: "); ViewObj(r); Print("\n");
+>    Print("PrintObj: ", r, "\n");
+>    Print("Display: "); Display(r);
+>    Print("DisplayString: ", DisplayString(r));
+>    Print("ViewString: ", ViewString(r), "\n");
+>    Print("PrintString: ", PrintString(r), "\n");
+>    Print("String: ", String(r), "\n");
+>  end;;
+
+#
+gap> TestPrintRangeRep([0..1]);
+ViewObj: [ 0, 1 ]
+PrintObj: [ 0, 1 ]
+Display: [ 0, 1 ]
+DisplayString: <object>
+ViewString: [ 0, 1 ]
+PrintString: [ 0, 1 ]
+String: [ 0, 1 ]
+gap> TestPrintRangeRep([0..2]);
+ViewObj: [ 0 .. 2 ]
+PrintObj: [ 0 .. 2 ]
+Display: [ 0 .. 2 ]
+DisplayString: <object>
+ViewString: [ 0 .. 2 ]
+PrintString: [ 0 .. 2 ]
+String: [ 0 .. 2 ]
+gap> TestPrintRangeRep([0,2..4]);
+ViewObj: [ 0, 2 .. 4 ]
```

Comment:
I guess we could go one step further and also print `[ 0, 2 .. 4 ]` as `[ 0, 2, 4 ]`...

Reply:
The input `[ 0, 2, 4 ]` does not produce an object in `IsRangeRep`, whereas `[ 0, 2 .. 4 ]` creates such an object.
In this sense, I would not be in favour of this change.

Reply:
Note that the documentation does not make any claim that the *representation* is preserved by reentering the output of Print; just that an equal object is returned. So both print outputs conform to the documentation.

In any case, always printing ranges as ranges certainly is also an option; the main motivation for this PR really is to make it behave consistently in one way or the other.

Reply:
I would support this kind of change, if you want to make it.